### ISSUE TITLE
[FEATURE] GUI trainer: Validate all files before starting

### ISSUE THREAD

**Issue by sdatkinson** [sdatkinson/neural-amp-modeler#344]
Warnings are good! 

Currently they are checked as they are trained, which can result in a problem in the middle of the night. Not good!

**Comment by sdatkinson** [comment]
Before doing this, it would be good to make some logging for the problems and simply note them instead of crashing. This should be a separate Issue that comes first.
